src/commands/playlist/playlist.command.ts

```diff
@@ -176,26 +214,33 @@ export class PlaylistCommand {
       );
     }
 
+    const paddingNumber = playlist.getLength() >= 100 ? 3 : 2;
+
     const content = chunk
       .map((track, index) => {
         const isCurrent = track === playlist.getActiveTrack();
 
-        // use the offset for the page, add the current index and offset by one because the array index is used
-        let point = `${offset + index + 1}. `;
-        point += `**${trimStringToFixedLength(track.name, 30)}**`;
-
+        let line = `\`\`${zeroPad(offset + index + 1, paddingNumber)}.\`\` `;
+        line += this.getTrackName(track, isCurrent) + ' • ';
         if (isCurrent) {
-          point += ' :loud_sound:';
+          line += lightFormat(track.getPlaybackProgress(), 'mm:ss') + ' / ';
         }
-
-        point += '\n';
-        point += Constants.Design.InvisibleSpace.repeat(2);
-        point += formatMillisecondsAsHumanReadable(track.getDuration());
-
-        return point;
+        line += lightFormat(track.getDuration(), 'mm:ss');
+        if (isCurrent) {
+          line += ' • (:play_pause:)';
+        }
+        return line;
       })
       .join('\n');
 
     return new EmbedBuilder().setTitle('Your playlist').setDescription(content);
   }
+
+  private getTrackName(track: Track, active: boolean) {
+    if (active) {
+      return `**${trimStringToFixedLength(track.name, 30)}**`;
```

Comment:
It would be easier to understand if you call the function before the if statement and use the variable result for both